### PR TITLE
Replace custom tooltips with native ones.

### PR DESCRIPTION
**Pull request by JuliaKirschenheuter**
Replace custom tooltips with native ones. Remove "vue-tooltip" from project.

Fixes https://github.com/nextcloud/notifications/issues/1410

### ToDo:
- [x] Add additional visual-hidden `<span>` for date

### CODE REVIEW

**Review by artonge — APPROVED** [review]

**Review by Pytal — APPROVED** [review]

File: src/Components/Notification.vue [hunk 1]
```diff
@@ -1,8 +1,9 @@
 <template>
 	<li class="notification" :data-id="notificationId" :data-timestamp="timestamp">
 		<div class="notification-heading">
+			<span class="hidden-visually">{{ absoluteDate }}</span>
 			<span v-if="timestamp"
-				v-tooltip.bottom="absoluteDate"
+				:title="absoluteDate"
```
Comment (nickvergessen) [inline src/Components/Notification.vue:6]: Bildschirmfoto vom 2023-01-27 10-49-40

Works

File: src/Components/Parameters/File.vue [hunk 2]
```diff
@@ -20,21 +20,16 @@
 -->
 
 <template>
-	<a v-tooltip.bottom="title"
+	<a :title="title"
```
Comment (nickvergessen) [inline src/Components/Parameters/File.vue:23]: Works
Bildschirmfoto vom 2023-01-27 10-49-47

**Review by nickvergessen — APPROVED** [review]

**Comment by nickvergessen** [comment]
Rebased to get #1433 and other CI prs in, so it is green in the end

**Pull request merged**